src/MetaCompilation.Analyzers/UnitTests/MetaCompilation.Analyzers.UnitTests.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>net46</TargetFramework>
+    <TargetFramework>net472</TargetFramework>
```

Comment:
Version is copied from master.

Reply:
After #3093 is merged, you can revert all the target framework changes in this pull request (they are not necessary since I updated the Microsoft.CodeAnalysis reference to target Roslyn 2.10.0 in this branch).

Reply:
I did the merge and when I revert the projects to `net46` I have a bunch of tests failing because they rely on APIs available only starting from `net47`. I am not sure whether that's best to stick with `net46` and update the failing tests to reference `net47` assemblies or to change/keep `net47` for test projects.